src/crypto/crypto.js

```diff
@@ -92,34 +92,33 @@ export async function publicKeyEncrypt(algo, publicParams, data, fingerprint) {
  * Decrypts data using specified algorithm and private key parameters.
  * See {@link https://tools.ietf.org/html/rfc4880#section-5.5.3|RFC 4880 5.5.3}
  * @param {module:enums.publicKey}        algo        Public key algorithm
- * @param {Object}                        publicKeyParams   Algorithm-specific public key parameters
- * @param {Object}                        privateKeyParams  Algorithm-specific private key parameters
+ * @param {Object}                        publicParams   Algorithm-specific public key parameters
+ * @param {Object}                        privateParams  Algorithm-specific private key parameters
  * @param {Array<module:type/mpi|
-                 module:type/ecdh_symkey>}
-                                          data_params encrypted session key parameters
+                 module:type/ecdh_symkey>}  data_params encrypted session key parameters
  * @param {Uint8Array}                    fingerprint Recipient fingerprint
  * @returns {Uint8Array}                  decrypted data
  * @async
  */
-export async function publicKeyDecrypt(algo, publicKeyParams, privateKeyParams, data_params, fingerprint) {
+export async function publicKeyDecrypt(algo, publicParams, privateParams, data_params, fingerprint) {
```

Comment:
I kind of like having `publicKeyParams` and `privateKeyParams` here because we also have `data_params` (which should probably really be called `messageParams` or `sessionKeyParams`) - those are also public, so it helps to disambiguate `publicKeyParams` I think. If you want to make it consistent with the other function, I would rename the param above to `publicKeyParams`, as well.

Reply:
Makes sense, I'll leave this for now